src/service/video/docking.js

```diff
@@ -803,30 +809,41 @@ export class VideoDocking {
    * @param {!../../video-interface.VideoOrBaseElementDef} video
    * @param {!RelativeX} posX
    * @param {!RelativeY} posY
-   * @param {boolean=} finalize
    * @private
    */
-  dock_(video, posX, posY, finalize = false) {
+  dockOnPositionChange_(video, posX, posY) {
     if (this.ignoreDueToDismissal_(video)) {
       return;
     }
 
-    const {element} = video;
-    const step = finalize ? 1 : this.calculateStep_(element);
-
+    const step = this.calculateStep_(video.element);
     if (this.ignoreDueToTransitionEnd_(step)) {
       return;
     }
 
+    this.dock_(video, posX, posY, step);
+
+    this.getDockingTimeout_().trigger(DOCKING_TIMEOUT, video);
+  }
+
+  /**
+   * @param {!../../video-interface.VideoOrBaseElementDef} video
+   * @param {!RelativeX} posX
+   * @param {!RelativeY} posY
+   * @param {?number=} optStep
+   * @private
+   */
+  dock_(video, posX, posY, optStep = null) {
```

Comment:
nit: the format for optional parameters used in most AMP code is `opt_variableName`

Reply:
Done.